src/tests.rs

```diff
@@ -523,27 +579,34 @@ fn floating_point_arithmetic_works() {
 #[test]
 fn reward_below_vesting_period_works() {
 	empty().execute_with(|| {
+		// The init relay block gets inserted
 		roll_to(2);
+		let init_block = Crowdloan::init_relay_block();
 		assert_ok!(mock::Call::Utility(UtilityCall::batch_all(vec![
-			mock::Call::Crowdloan(crate::Call::initialize_reward_vec(
-				vec![([4u8; 32].into(), Some(1), 1247)],
-				0,
-				3
-			)),
-			mock::Call::Crowdloan(crate::Call::initialize_reward_vec(
-				vec![([5u8; 32].into(), Some(2), 1247)],
-				1,
-				3
-			)),
+			mock::Call::Crowdloan(crate::Call::initialize_reward_vec(vec![(
+				[4u8; 32].into(),
+				Some(1),
+				1247
+			)])),
+			mock::Call::Crowdloan(crate::Call::initialize_reward_vec(vec![(
+				[5u8; 32].into(),
+				Some(2),
+				1247
+			)])),
 			// We will work with this. This has 5/8=0.625 payable per block
-			mock::Call::Crowdloan(crate::Call::initialize_reward_vec(
-				vec![([3u8; 32].into(), Some(3), 6)],
-				2,
-				3
-			))
+			mock::Call::Crowdloan(crate::Call::initialize_reward_vec(vec![(
```

Comment:
all these calls are public so I don't think you have to do wrap the inner call in the outer call

I expect it would work with just `Crowdloan::initialize_reward_vec(...)`

Reply:
The compiler is telling me I need an outer call type inside batch_all 

`expected enum `mock::Call`, found enum `pallet::Call`